src/ispc.cpp

```diff
@@ -1334,10 +1334,13 @@ std::string Target::GetTripleString() const {
     case TargetOS::macos:
         // asserts
         if (m_arch != Arch::x86_64) {
```

Comment:
Fix for correct selection of macOS arch.
```suggestion
        if (m_arch == Arch::x86_64) {
```